broker/lib/fabrik/DirectorTaskPoller.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const bosh = require('../bosh');
+const DirectorManager = require('./DirectorManager');
+const logger = require('../logger');
+const catalog = require('../models/catalog');
+const pubsub = require('pubsub-js');
+const CONST = require('../constants');
+const config = require('../config');
+const boshCache = bosh.BoshOperationCache;
+const TIME_POLL = 1 * 60 * 1000;
+const LockStatusPoller = require('./LockStatusPoller');
+
+class DirectorTaskPoller extends LockStatusPoller {
+  constructor() {
+    super({
+      time_interval: TIME_POLL
+    });
+  }
+
+  action() {
+    return boshCache.getDeploymentNames().mapSeries(deploymentName => {
+        return Promise.try(() => {
+            return boshCache.getDeploymentByName(deploymentName);
+          })
+          .then(cached => {
+            let catalogPlan = catalog.getPlan(cached.plan_id);
+            return DirectorManager.load(catalogPlan).createOrUpdateDeployment(deploymentName, cached.params, cached.args);
+          })
+          .catch(e => {
+            logger.error(`Error in automated deployment for ${deploymentName}`, e);
```

Comment:
automated deployment?

Reply:
wanted to convey `error in scheduled deployment operation for ... `
I guess it came out wrong. Will change